source/views/directory/types.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+export type DirectoryType = {
+	displayName: string,
+	classYear: number,
+	suffixName: string,
+	extension: number,
+	firstName: string,
+	thumbnail: URL,
+	departments: Array<DepartmentType>,
+	email: string,
+	homePhone: null,
+	onLeave: boolean,
+	officePhone: string,
+	homeAddress: {
+		zip: string,
+		city: string,
+		country: string,
+		state: string,
+		street: Array<string>,
```

Comment:
not sure why this is an array on the backend?

Reply:
Because there are up to two street addresses and I didn't want to put two props in